frontend/packages/kubevirt-plugin/integration-tests-cypress/views/selector.ts

```diff
@@ -24,8 +24,8 @@ export enum dashboardTab {
   detailsCardItem = '.co-details-card__item-value',
   utilsCardItem = '[data-test-id="utilization-item"]',
   inventoryCardItem = '.co-inventory-card__item',
-  vmHealth = '[data-test="Virtual Machine-health-item"]',
-  guestAgentHealth = '[data-test="Guest Agent-health-item"]',
+  vmHealth = '[data-item-id="Virtual Machine-health-item"]',
```

Comment:
I'm sorry. I must have missed this in my last review. The convention for data* attributes is all lowercase and with words separated by hyphens. So, `Virtual Machine-health-item` would be `virtual-machine-health-item`.

The bigger issue is that I don't see any element assigned this ID in the code, so these selectors shouldn't be returning anything.

Reply:
Make changes to below two places can fix it, but I'm not sure whether it's safe to do it.

https://github.com/openshift/console/blob/c6923b9c593158114500a43683fa10c381e0a9c8/frontend/packages/kubevirt-plugin/src/components/dashboards-page/vm-dashboard/status/GuestAgentStatusHealth.tsx#L49
https://github.com/openshift/console/blob/c6923b9c593158114500a43683fa10c381e0a9c8/frontend/packages/kubevirt-plugin/src/components/dashboards-page/vm-dashboard/status/VMStatusHealth.tsx#L23

Above codes combine with below shared code generate the data* attributes.
https://github.com/openshift/console/blob/c6923b9c593158114500a43683fa10c381e0a9c8/frontend/packages/console-shared/src/components/dashboard/status-card/HealthItem.tsx#L24

Reply:
Just find out it cannot be simply replaced as it changes the string shows in dashboard status card as well.
So I'll leave it there.

Reply:
Ah. I'm not a fan of that. I think it would have been better to accept a dataTest prefix prop. Just leave it as it is for now. =)